goreleaser/docker-build-pipelines.yml

```diff
@@ -0,0 +1,50 @@
+dist: /tmp/pachyderm/release/docker-build-pipeline
+
+before:
+  hooks:
+    - docker login
+
+builds:
+  -
+    id: pachctl
+    dir: src/server/cmd/pachctl
+    main: main.go
+    binary: pachctl
+    ldflags:
+      - -X {{ .Env.LD_FLAGS }} -X main.commit={{.Commit}} -X main.date={{.Date}} -X main.builtBy=goreleaser
+    gcflags:
+      - all=-trimpath={{ .Env.PWD }}
```

Comment:
Apparently to do this right there also needs to be asmflags with a similar trimpath directive?  Maybe unnecessary since we set CGO_ENABLED=0.